### ISSUE TITLE
code about validation set

### ISSUE THREAD

**Issue by fengxiaohu** [xueleichen/PyTorch-Underwater-Image-Enhancement#6]
I have a qusetion about validation set.
In my opinion dataset usually divide dataset into training_set and validation_set.
accdoring your paper,the dataset was divided into two part(700 images for training and the rest for test)
Therefore I check the code.the uwcc.py which is override for DataSet.
would you mind telling me which code for selecting validation_set?
